staking/programs/staking/src/state/vesting.rs

```diff
@@ -25,6 +25,11 @@ pub enum VestingSchedule {
         period_duration: u64,
         num_periods:     u64,
     },
+    PeriodicVestingAfterListing {
+        initial_balance: u64,
+        period_duration: u64,
+        num_periods:     u64,
+    }
```

Comment:
Wondering if it's better to keep PeriodicVesting , but deprecate the start_data field

Reply:
We may need both. We have tokens that are subject to lockups post-token-list, and tokens that are subject to lockups with fixed date schedules.